src/jdk.jlink/share/classes/jdk/tools/jlink/internal/ExecutableImage.java

```diff
@@ -61,4 +61,25 @@
      * @param args Additional arguments
      */
     public void storeLaunchArgs(List<String> args);
+
+    /**
+     * Set the Platform of the image.
+     *
+     * @param p
+     */
+    public void setTargetPlatform(Platform p);
+
+    /**
+     * The Platform of the image.
+     *
+     * @return Platform
+     */
+    public Platform getTargetPlatform();
+
+    /**
+     * Checks if the image is 64-bit.
+     *
+     * @return boolean
+     */
+    public boolean is64Bit();
```

Comment:
This `is64Bit` method should belong to `Platform` class

Reply:
Done. I've moved the method to the `Platform` class.